ensembl-track-api.openapi.yaml

```diff
@@ -34,25 +34,60 @@ paths:
           required: true
           description: Stable genome ID.
           schema:
-            type : string
+            type: string
+            format: uuid
           example: a7335667-93e7-11ec-a39d-005056b38ce3
       responses:
         '200':
           description: Successful request.
           content:
             application/json:
               schema:
-                type: object
-                properties:
-                  track_categories:
-                    type: array
-                    items:
-                      $ref: '#/components/schemas/TrackCategory'
+                $ref: '#/components/schemas/TrackCategories'
         '400':
           description: Invalid request.
         '404':
           description: Specified genome ID was not found.
-  /tracks/{track_id}:
+  /track:
+    post:
+      summary: Creates a new track.
+      requestBody:
+        description: Data for the new track.
+        required: true
+        content:
+          application/json:
+            schema:
+              allOf:
+                - $ref: '#/components/schemas/Track'
+                - $ref: '#/components/schemas/TrackDatafiles'
+                - $ref: '#/components/schemas/TrackDescription'
+      responses:
+        '201':
+          description: Track was successfully created.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  track_id:
+                    description: UUID of the created track.
+                    type: string
+                    format: uuid
+                    example: d0df738a-0ecb-4b1e-8576-a5621a4b15d2
+        '400':
+          description: Invalid request.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    description: >
+                      Error message (for example, if a track with the same 
+                      genome ID, label and datafiles already exists).
+                    type: string
+                    example: Track already exists.
+  /track/{track_id}:
```

Comment:
So this means that the whole url will be `/api/tracks/track/:track_id`? Interesting...

Reply:
Yes, because the root `/api/tracks/` is set by k8s. The alternatives are `/api/tracks` (empty path ok?) `/api/tracks/tracks` (not good, but currently in use).